docs/_plugins/shortcodes/rh-playground.js

```diff
@@ -0,0 +1,135 @@
+import { LitElement, html, css } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+
+import 'playground-elements';
+import '@rhds/elements/rh-button/rh-button.js';
+import '@rhds/elements/rh-spinner/rh-spinner.js';
+
+class RhPlayground extends LitElement {
+  static styles = css`
+    :host {
+      max-height: 785px;
```

Comment:
Let's remove this max-height rule while we are at it.  This stops the element from being able to be resized vertically. 

```Suggestion
```

Reply:
@marionnegp this was to prevent long demos from taking over the page

maybe max height + resizable?

Reply:
fwiw I did the change locally, each subcomponent of playground has a height, I don't think the outer host can scale beyond the sum of those parts. (tab bar, editor, preview) Doesn't really look like this rule is doing anything in its current state.

Reply:
You might be thinking of the next rule down for the `div` which also sets the same max-height which is the pre-load demo view.  That would allow long demos to take over.

Reply:
ok whichever you see fit :) feel free to commit directly to the branch